bigbluebutton-tests/playwright/core/page.js

```diff
@@ -304,6 +304,10 @@ class Page {
         await helpers.sleep(1000);  // expected time to toast notification disappear
       }
   }
+
+  async setViewPortSize(width, height) {
+    await this.page.setViewportSize({ width, height});
+  }
```

Comment:
I prefer to use it as an object as we see clearly what is each value declared without the need of going to the function

```suggestion
  async setViewPortSize({ width, height }) {
    await this.page.setViewportSize({ width, height});
  }
```